apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -40,6 +40,53 @@
             <tr><th>Instance</th><td colspan="2"><%= assessment_instance.number %>
                 (<a href="<%= plainUrlPrefix %>/course_instance/<%= course_instance.id %>/assessment_instance/<%= assessment_instance.id %>">student view</a>)
             </td></tr>
+            <% if (instance_user) { %>
+            <tr>
+              <th>Fingerprint Changes</th>
```

Comment:
Also we should add a news item. Or are we switching to blog posts, @nwalters512 ?

Reply:
Don't worry about this for now.